torch/_utils_internal.py

```diff
@@ -121,6 +121,13 @@ def justknobs_check(name: str) -> bool:
     return True
 
 
+def justknobs_getval_int(name: str) -> int:
+    """
+    Read warning on justknobs_check
+    """
+    return 0
```

Comment:
Are we doing some weird monkeypatching in fbcode?  Let's not do that, it makes things hard to follow.  You can create a `/fb/` folder that won't get exported to github.

Reply:
I did not add any monkey patching, there's an fb version of this file. This file contains the stubs. Please look at D54818197

Reply:
I followed @ezyang 's example from https://github.com/pytorch/pytorch/pull/120915

Reply:
We selectively include fb/_utils_internal.py in fbcode vs torch/_utils_internal.py in open source.  It's not monkeypatching exactly but it is a bit unusual.  I think @oulgen is just following the pattern for JK killswitches introduced by @ezyang above.

Reply:
(Oh ha you said exactly the same thing in the span of time it took me to write my comment :-p )